dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/MetricsGlobalRegistry.java

```diff
@@ -17,13 +17,18 @@
 
 package org.apache.dubbo.metrics;
 
+import io.micrometer.core.instrument.Metrics;
 import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
 
 public class MetricsGlobalRegistry {
 
     private static final CompositeMeterRegistry compositeRegistry = new CompositeMeterRegistry();
 
     public static CompositeMeterRegistry getCompositeRegistry() {
-        return compositeRegistry;
+        if (Boolean.parseBoolean(System.getProperty("dubbo.metrics.useGlobalRegistry", "true"))) {
```

Comment:
Try to get from `ApplicationModel.environment`

Reply:
0